src/main/resources/google_checks.xml

```diff
@@ -109,26 +109,26 @@
         </module>
         <module name="ParameterName">
             <property name="id" value="ParameterNameNonPublic"/>
-            <property name="format" value="^[a-z]([a-z0-9][a-zA-Z0-9]*)?$"/>
+            <property name="format" value="^[a-z]([a-z0-9]+[A-Z]?)*$"/>
             <property name="accessModifiers" value="protected, package, private"/>
             <message key="name.invalidPattern"
              value="Parameter name ''{0}'' must match pattern ''{1}''."/>
         </module>
         <module name="ParameterName">
             <property name="id" value="ParameterNamePublic"/>
-            <property name="format" value="^[a-z][a-z0-9][a-zA-Z0-9]*$"/>
+            <property name="format" value="^[a-z]([a-z0-9][A-Z]?)+$"/>
```

Comment:
I am not sure why non-public and public regexps are different. Google confirmed two times in issue on google style that checkstyle rule should be relaxed. It is their style - so we follow it. If smb do not like this (I hope most of projects) they will change this in their configuration.

`accessModifiers` should be removed from google style config.